examples/js/loaders/FBXLoader2.js

```diff
@@ -1212,6 +1212,11 @@
 									material = new THREE.MeshBasicMaterial( { color: 0x3300ff } );
 
 								}
+								if ( 'color' in geometry.attributes && geometry.attributes.color.count > 0 ) {
+
+									material.vertexColors = THREE.VertexColors;
```

Comment:
You might need to iterate through the array of materials here instead of setting the vertexColors property on the Multi-Material (If there is a multi-material).  The code here admittedly is a little inconsistent, I might go ahead and make a quick fix.